.github/workflows/native-image.yml

```diff
@@ -12,17 +16,32 @@ jobs:
         with:
           java-version: '21.0.1'
           distribution: 'graalvm'
+          cache: 'maven'
           github-token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build native image
+      - name: Build develop native image
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
         run: >
           mvn -B clean package -Pnative
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
```

Comment:
only required for SonarCloud

Reply:
removed in https://github.com/cryptomator/hub-cli/pull/5/commits/370686fd2594bf8b97753bca28b90c616df79140